src/main/kotlin/de/livepoll/api/entity/dto/UserDto.kt

```diff
@@ -1,18 +1,7 @@
 package de.livepoll.api.entity.dto
 
-import com.sun.istack.NotNull
-
 data class UserDto(
-        @NotNull
```

Comment:
The user id would be helpful for the frontend. It will be required to provide the user id to retrieve e.g. all polls for that particular user.